test/lib/generate-layouts.spec.js

```diff
@@ -5,8 +5,13 @@ var expect = require("chai").expect;
 // Strict mode leads to odd bug in Node < 0.12
 var mockRequire = require("mock-require");
 
+// to ensure cross-platform consistency with mixed Posix & Win32 paths, use normalize()
+// ideally, this would be taken care of as a fix for mockRequire.
+// see https://github.com/boblauer/mock-require/issues/20
+var normalize = require("path").normalize;
+
 var mock = function (path, obj) {
-  return mockRequire(process.cwd() + "/" + path, obj);
+  return mockRequire(normalize(process.cwd() + "/" + path), obj);
```

Comment:
The existing string concat with `"/"` feels wrong.

Wouldn't 

```js
return mockRequire(path.resolve(path), obj);
```

solve all the problems we have here?